tests/device_tests/test_msg_sd_protect.py

```diff
@@ -31,6 +31,11 @@ def test_sd_protect(self, client):
         with pytest.raises(TrezorFailure):
             device.sd_protect(client, proto.SdProtectOperationType.DISABLE)
 
+        # skip the rest of the tests if SD card is not present
+        debug_state = client.debug.state()
+        if not debug_state.sd_card:
```

Comment:
I am wondering whether we should add something like `@pytest.mark.skip_no_sd_card` instead of having `if not ... return` inside the test. Just a thought. I leave it to you and @matejcik to decide how this should be done.

Reply:
We could use `pytest.skip` I guess. I don't have a strong opinion on this, let's see what @matejcik thinks.

Reply:
Question is whether it's easy in its current form to access the `client` object withing the `@pytest.mark.skip_....`. If it's easy, let's do it now.

Reply:
skipping is a much better option than a warning. i've added a `pytest.mark.sd_card` marker.

Reply:
Great :+1: .